fsharp-backend/src/BackendOnlyStdLib/LibStaticAssets.fs

```diff
@@ -151,8 +150,6 @@ let fns : List<BuiltInFn> =
         (function
         | state, [] ->
           uply {
-            // CLEANUP calling this with no deploy hash generates an error
-            // (should be Option<TStr>)
```

Comment:
Ah - should I change this to Option, actually?

Reply:
Yes! (deprecate + new, obvi)

Reply:
Yes,  I think an option rather than a result